accounts-cluster-bench/src/main.rs

```diff
@@ -692,6 +701,9 @@ fn run_accounts_bench(
         if iterations != 0 && count >= iterations {
             break;
         }
+        if max_accounts_met {
+            break;
```

Comment:
nit: add a log that we reached max account and exiting?

Reply:
Added (last commit). No other changes; just rebased to pick up the local-cluster fix